src/lib/plugins/BrowserPlugin/BrowserPlugin.ts

```diff
@@ -40,6 +42,19 @@ export function createBrowserPlugin({history}: BrowserPluginProps): PluginConstr
     @exportDependency(HOST)
     host = window.location.host
 
+    @exportDependency(BASE_URL)
+    baseUrl = this.constructBaseUrl()
+
+    constructBaseUrl(): string {
+      let baseUrl = window.location.protocol + '//' + window.location.hostname
+      if (window.location.port && window.location.port !== '' &&
```

Comment:
Could this be refactored out into a portIsntHttpOrHttps method?

Reply:
Done